go/libkb/secret_store_secretservice.go

```diff
@@ -58,8 +62,28 @@ func (s *SecretStoreSecretService) maybeRetrieveSingleItem(mctx MetaContext, srv
 	return &item, nil
 }
 
-func (s *SecretStoreSecretService) RetrieveSecret(mctx MetaContext, username NormalizedUsername) (secret LKSecFullSecret, err error) {
-	defer mctx.TraceTimed("SecretStoreSecretService.RetrieveSecret", func() error { return err })()
+func (s *SecretStoreRevokableSecretService) ekstoreDir(mctx MetaContext, username string) string {
```

Comment:
ek usually refers to `ephemeralKey`throughout the codebase, could you change the naming up here?